README.md

```diff
@@ -1,2 +1,160 @@
-# ROS_WBMPC
-ROS interface for the Whole Body Model Predictive Controller
+[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/loco-3d/linear-feedback-controller/main.svg)](https://results.pre-commit.ci/latest/github/loco-3d/linear-feedback-controller/main)
+
+# linear feedback controller
+
+This repository contains two packages:
+- the [linear_feedback_controller](./linear_feedback_controller/README.md)
+- the [linear_feedback_controller_msgs](./linear_feedback_controller_msgs/README.md)
+
+These two package are under the license [BSD-2 license](./LICENSE).
+
+The current versions of this 2 package can be found in their respective `package.xml` file.
+
+Follows a quick description of these package.
+
+## The linear_feedback_controller
+
+In this package we implement a [roscontrol](http://wiki.ros.org/ros_control)
+controller. It is based on the [pal_base_ros_controller](https://gitlab.com/pal-robotics/LAAS/pal_base_ros_controller_tutorials)
+package.
+
+We implement a RosTopic publisher that sends the state of the robot:
+- base configuration
+- base velocity
+- joint positions
+- joint velocities
+- joint efforts (torques applied to the joints)
+
+We then subscribe to a RosTopic that sends us controls for the robot:
+- A feedback gain matrix
+- A feedforward term in torque
+- The state which was used to linearize the control.
+
+This allows us, for example,to use it on the Talos (1 and 3) robots with a remote controller
+using a whole body model predictive control based on [croccodyl](https://github.com/loco-3d/crocoddyl)
+
+Please check the [README.md](./linear_feedback_controller/README.md) of the package for more details.
+
+## The linear_feedback_controller_msgs
+
+This package contains the external user interface to the linear_feedback_controller
+package. It describes the sensor data exchanged in the previously cited RosTopics.
+
+And in particular it offers a very simple ROS/[Eigen](https://eigen.tuxfamily.org/index.php?title=Main_Page)
+conversion. This is made to facilitate further computations with the Sensor
+data. And ease to fill in the Control message.
+
+Please check the [README.md](./linear_feedback_controller_msgs/README.md) of the package for more details.
+
+## Example of usage
+
+### Requirements
+
+For this example you need to have access to the dockers in https://gitlab.laas.fr/gsaurel/docker-pal/.
+
+### Build the repository using ROS and docker.
+
+First of all build or pull the docker from https://gitlab.laas.fr/gsaurel/docker-pal/
+using the branch `gsaurel`.
```

Comment:
These are public instructions explaining the need to download private stuff. We need either to make everything public, or make everything private, or explain that this can't be done outside of the lab for now

Reply:
All of this is public if I am not mistaken, docker/simulation included. There are no private header if I recall.

Reply:
https://gitlab.laas.fr/gsaurel/docker-pal is private. You can try the docker images associated to this project without beeing logged in: it won't work

Reply:
ah! But I guess that we need to create a public docker then at least for CI purposes...
We could base the image from the official PAL Talos simulation image.